fastlane/jetpack_metadata/en-US/release_notes.txt

```diff
@@ -1,6 +1,12 @@
-Made a New Year's resolution to publish more content on your site? We made a few updates to the Block Editor to help you do just that.
+What time is it? Our brand-new time zone selection screen knows. Get a suggested time zone based on your device and improve search while you’re at it.
 
-- When you're editing Paragraph blocks, you can highlight specific text in any color you'd like.
-- Brand-new Gallery blocks will default to the new design. You can also auto-convert any old galleries to the new format without a hitch.
-- The editor now auto-scrolls, so you can always see what you're typing. No more disappearing text, no more guesswork.
-- We squashed a bug where the background color wasn't showing up for pre-formatted blocks on sites that use standard themes.
+If you’re using a free plan, video uploads are now limited to five minutes per file. Don’t worry, that won’t affect any videos you’ve already uploaded to your site.
+
+We’ve also made some changes to the block editor:
+
+- We solved translation issues that caused some user interface elements (like text alignment information and the block editor modal window) to only appear in English.
+- We added language translation for Jetpack and Layout Grid blocks.
+- We fixed a bug that would reset text formatting to default after hitting backspace. Bold on, friends.
+- Multi-line block titles are now center-aligned in the inserter menu.
+
+Finally, the “Share WordPress with a friend” row is back where it belongs on the Me screen. Reunited and it feels so good.
```

Comment:
As the diff shows, this file existed already. Clearly, the change was not required to get `deliver` to recognize the language for metadata upload.

I made it because GlotPress doesn't currently have an `en-US` locale to fetch, meaning the scripts that downloads the localized metadata won't update the English release notes. This will need to be fixed in a dedicated PR.

In the meantime, this manual change will ensure the release notes are correct for 19.0.